### DISCUSSION TITLE
script not woring properly

### DISCUSSION THREAD

**Discussion by iamLazyCode** [moudey/Shell#332] [General]
hi @moudey I have a script that copies font file from one place to another when I run the script alone it works but if I add it to the shell contect menu and run the script the font doesn't get copied...this is the command I used

```
item(title='Conversion' image=\uE1ED pos=1 cmd args='/k "C:\Users\PitchAbyss\OneDrive - MSFT\My-All-Setup-Guides\01 Random Guides\02 My Scripts\Anime Scripts\My Final Scripts\Final_Script_For Media_Conversion.bat"')
```

also is it possible to replace cmd with latest PowerShell core, please reply

**Comment by iamLazyCode** [comment]
@moudey sir please reply

**Comment by moudey** [comment]
> @moudey sir please reply

My children and I have been detained in the hospital for 4 days. When I get out, I will check your issue.

**Reply by iamLazyCode** [reply]
I am extremely sorry sir @moudey dint knew please tc of everyone and check once time allows

**Comment by moudey** [comment]
The script may need to change the current folder to the path of the script folder
```
item(title='Conversion' image=\uE1ED pos=1 
    cmd args='/k "C:\Users\PitchAbyss\OneDrive - MSFT\My-All-Setup-Guides\01 Random Guides\02 My Scripts\Anime Scripts\My Final Scripts\Final_Script_For Media_Conversion.bat"'

    dir="C:\Users\PitchAbyss\OneDrive - MSFT\My-All-Setup-Guides\01 Random Guides\02 My Scripts\Anime Scripts\My Final Scripts")
```

**Comment by iamLazyCode** [comment]
@moudey 

thankyou for reply sir and i hope everything is fine now 

sir this command simply stops the shell and takes context menu to stock

![image](https://github.com/moudey/Shell/assets/13099434/f6f92b86-23aa-44dc-9f38-d9e5946084b6)

**Comment by moudey** [comment]
Sorry. Replace the double quotation mark with a single quotation mark with the path in the `dir` property

```
item(title='Conversion' image=\uE1ED pos=1 
    cmd args='/k "C:\Users\PitchAbyss\OneDrive - MSFT\My-All-Setup-Guides\01 Random Guides\02 My Scripts\Anime Scripts\My Final Scripts\Final_Script_For Media_Conversion.bat"'

    dir='C:\Users\PitchAbyss\OneDrive - MSFT\My-All-Setup-Guides\01 Random Guides\02 My Scripts\Anime Scripts\My Final Scripts')
```

**Comment by iamLazyCode** [comment]
> Sorry. Replace the double quotation mark with a single quotation mark with the path in the `dir` property
> 
> ```
> item(title='Conversion' image=\uE1ED pos=1 
>     cmd args='/k "C:\Users\PitchAbyss\OneDrive - MSFT\My-All-Setup-Guides\01 Random Guides\02 My Scripts\Anime Scripts\My Final Scripts\Final_Script_For Media_Conversion.bat"'
> 
>     dir='C:\Users\PitchAbyss\OneDrive - MSFT\My-All-Setup-Guides\01 Random Guides\02 My Scripts\Anime Scripts\My Final Scripts')
> ```

thankyou so much sir it did sort the earlier issue but still doest copies the font file to the directory

if u want i can make a screen capture

**Reply by moudey** [reply]
Please attach "Final_Script_For Media_Conversion.bat"

**Comment by iamLazyCode** [comment]
> Please attach "Final_Script_For Media_Conversion.bat"

please check mail sir , also enjoy a little coffe :)

![image](https://github.com/moudey/Shell/assets/13099434/f6085fca-430b-4e0e-990b-9fc6d1e1d484)
